coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -2665,6 +2665,40 @@ def test_clamp(self, backend, shape, clamp_range):
             shape, model, backend=backend, rand_range=(-5, 5)
         )
 
```

Comment:
I don't think the test belongs to `Testelementwise`, please create a new test class like `TestTriu`